src/routes.js

```diff
@@ -3,12 +3,16 @@
 /* eslint-disable import/no-named-as-default */
 /* eslint-disable no-unused-vars */
 import React from 'react';
-import {
-  Router, Route as DefaultRoute, Switch, Redirect
-} from 'react-router-dom';
+import { Router, Route as DefaultRoute, Switch, Redirect } from 'react-router-dom';
+
 import history from './history';
-import SignInPage from './components/Screens/Auth/SignIn';
 import LandingPage from './containers/LandingPage';
+import AgentSignUp from './components/Screens/Auth/AgentSignUp';
+import NavBar from './components/Navigations/NavBar/index';
+import AdminLogin from './components/Screens/Auth/AdminLogin';
+import CustomerDashboard from './components/Screens/Auth/CustomerDashboard';
+import Signup from './components/Screens/Auth/Signup';
+import Profile from './components/Screens/Auth/Profile';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "./components/Screens/Auth/Profile"](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)